app/grandchallenge/algorithms/templates/algorithms/partials/socket_modal_base.html

```diff
@@ -0,0 +1,74 @@
+{% load bleach %}
+{% load json %}
+
+
+<a href="#socketModal-{{ socket.pk }}"
+   data-toggle="modal"
+   data-target="#socketModal-{{ socket.pk }}"
+   class="text-decoration-none"
+   role="button"
+   title="View {{ socket.title }}"
+>
+    <div class="badge badge-info mr-1">
+        <i class="fas fa-fw fa-info"></i>
+    </div>
+</a>
+
+<div class="modal" id="socketModal-{{ socket.pk }}" tabindex="-1" role="dialog"
+     aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered modal-lg" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title">
+                    {{ socket.title }}
+                </h5>
+                <button type="button" class="close" data-dismiss="modal"
+                        aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <dd class="text-left modal-body">
+                <dl>
+                    <dt>Description</dt>
+                    <dd>{% if socket.description %}{{ socket.description }}{% else %} - {% endif %}</dd>
+
+                    <dt>Kind</dt>
+                    <dd>{{ socket.get_kind_display }}</dd>
+
+                    {% if input %}
+                        <dt>Read from</dt>
+                        <dd>
+                            {% if socket.is_image_kind %}
+                                <code>/input{% if socket.relative_path %}/{{ socket.relative_path }}{% endif %}/&lt;uuid&gt;.mha</code> or
+                                <code>/input{% if socket.relative_path %}/{{ socket.relative_path }}{% endif %}/&lt;uuid&gt;.tif</code>
+                            {% else %}
+                                <code>/input/{{ socket.relative_path }}</code>
+                            {% endif %}
+                        </dd>
+                    {% else %}
+                        <dt>Write to</dt>
+                        <dd>
+                            {% if socket.is_image_kind %}
+                                <code>/output{% if socket.relative_path %}/{{ socket.relative_path }}{% endif %}/&lt;uuid&gt;.mha</code> or
+                                <code>/output{% if socket.relative_path %}/{{ socket.relative_path }}{% endif %}/&lt;uuid&gt;.tif</code>
+                            {% else %}
+                                <code>/output/{{ socket.relative_path }}</code>
+                            {% endif %}
+                        </dd>
+                    {% endif %}
```

Comment:
This seems duplicate of the other table, can a partial be created and it be reused?